### ISSUE TITLE
OpenApi 3.0 support to mimic v2 usage style

### ISSUE THREAD

**Issue by austek** [Swagger2Markup/swagger2markup#393]
The current implementation, Swagger v2 support, is used
```
Swagger2MarkupConverter.from(localSwaggerFile)
        .build()
        .toFile(outputFile);
```

Ideally, OpenApi 3.0 support #340 should have similar way to consume it as well, for example
```
Swagger2MarkupConfig config = new Swagger2MarkupConfigBuilder() 
        .withParseOptions(new ParseOptions()) 
        .build(); 
Swagger2MarkupConverter converter = Swagger2MarkupConverter.from(localSwaggerFile)
        .withConfig(config) 
        .build();
```
This would allow less documentation and seamless transition for any consumer/user.

**Comment by balajiv113** [comment]
@austek I would like to work on this, 
one question on this, are we going to change swagger 2 implementation also to the new one (asciidoc api) ?? Are the old one will still be there ?? 

If we are going to migrate to the new one then we can have a abstract configuration builder for both the versions (Like group, orderBy and even the extension registry) which will be common for both the versions

**Comment by austek** [comment]
Swagger 2 implementation is still supported. ideally, we use the same config for both versions, as the plan is to support the same group, orderBy and extensions. if not possible to use the same version abstraction will be the way to go.

Until there is a Markdown converter for asciidoc, asciidoc output only will be support.

Maybe create a new builder `OpenApi2MarkupConverter`

**Comment by austek** [comment]
Addressed by #397
